src/DnsHelperUI/MainForm.Designer.cs

```diff
@@ -236,13 +223,35 @@ private void InitializeComponent()
             this.btnChooseDns.UseVisualStyleBackColor = true;
             this.btnChooseDns.Click += new System.EventHandler(this.btnChooseDns_Click);
             // 
+            // cmbNics
+            // 
+            this.cmbNics.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
+            this.cmbNics.FormattingEnabled = true;
+            this.cmbNics.Location = new System.Drawing.Point(15, 13);
+            this.cmbNics.Name = "cmbNics";
+            this.cmbNics.Size = new System.Drawing.Size(278, 21);
+            this.cmbNics.TabIndex = 9;
+            this.cmbNics.SelectedIndexChanged += new System.EventHandler(this.cmbNics_SelectedIndexChanged);
+            // 
+            // btnShowConnections
+            // 
+            this.btnShowConnections.Location = new System.Drawing.Point(299, 13);
+            this.btnShowConnections.Name = "btnShowConnections";
+            this.btnShowConnections.Size = new System.Drawing.Size(75, 23);
+            this.btnShowConnections.TabIndex = 10;
+            this.btnShowConnections.Text = "Connections";
```

Comment:
Could you please make this button smaller and update the tooltip. Something along the lines of:

```
this.btnShowConnections.Location = new System.Drawing.Point(344, 10);
this.btnShowConnections.Name = "btnShowConnections";
this.btnShowConnections.Size = new System.Drawing.Size(30, 23);
this.btnShowConnections.TabIndex = 10;
this.btnShowConnections.Text = "...";
this.toolTip1.SetToolTip(this.btnShowConnections, "Open the Windows Network Settings Dialog");
```